addons/ballistics/CfgWeapons.hpp

```diff
@@ -51,12 +51,15 @@ class CfgWeapons {
     class DMR_03_base_F: Rifle_Long_Base_F {
         ACE_barrelLength = 508.0;
         ACE_barrelTwist = 254.0;
-        initSpeed = -0.991536;
         magazineWell[] += {
             "CBA_762x51_M14"
         }; // vanilla magazineWell[] = {"M14_762x51"};
     };
 
+    class srifle_DMR_03_F: DMR_03_base_F {
+        initSpeed = -0.991536; // Default vanilla value 940
```

Comment:
No, an explanation why this is here. And also keep the original change.

Reply:
I don't understand what do you mean with an explanation. There is no explanation with the old config.
>And also keep the original change
Same, i don't understand my mistake.